src/components/Profile/Profile.css

```diff
@@ -1,9 +1,21 @@
 .user-profile {
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    justify-content: space-evenly;
     align-items: center;
-    overflow: scroll;
     height: 400px;
     padding-top: 15%;
+    position: relative;
+}
+
+.user-profile > a > button {
+    position: absolute;
```

Comment:
Absolute positioning in a relatively positioned element might be my favorite CSS hack